tasks/main.yaml

```diff
@@ -71,7 +108,7 @@
     - name: Delete debug file
       file:
         path: "{{ logging_role_path|d('logging') }}/debug/main.yaml"
-        state: absent
+        state: 'absent'
```

Comment:
Why this change? Examples of Ansible module usage seem to never enclose such short strings in quotes (applies to many other places like `'latest'` above). See https://docs.ansible.com/ansible/latest/modules/file_module.html . Please do not deviate from examples in docs or existing code unnecessarily.